controllers/hibp.js

```diff
@@ -37,9 +37,9 @@ async function notify (req, res) {
     }
   }
 
-  if (breachAlert.IsSpamList || breachAlert.Domain === "" || breachAlert.IsFabricated) {
+  if (breachAlert.IsSpamList || breachAlert.Domain === "" || breachAlert.IsFabricated || !breachAlert.IsVerified) {
     log.info(`${breachAlert.Name} is fabricated, a spam list, or not associated with a website. \n Breach Alert not sent.`);
-    return res.json(
+    return res.status(200).json(
```

Comment:
@groovecoder or @nhnt11 Sorry, I know you guys are on PTO but do we need to give Troy a heads up that he might be seeing a new response from us?

Reply:
We probably should. Also, if the API calls for sending back the number of notified subscribers, we should send a zero value to indicate we didn't notify anyone, no?

Reply:
Great point, will update.